src/Projection/PdoEventStoreProjector.php

```diff
@@ -551,16 +551,14 @@ public function run(bool $keepRunning = true): void
                 }
 
                 if ($gapDetected && $this->gapDetection) {
-                    $sleep = $this->gapDetection->getSleepForNextRetry();
-
-                    \usleep($sleep);
+                    \usleep($this->gapDetection->getSleepForNextRetry() * 1000);
```

Comment:
I'm thinking about whether we should do the calculation or change the documentation and tell people that sleep time is configured in microseconds :thinking: 

When multiplying sleep time 1 ms is the shortest sleep interval possible. Maybe that's too long for some use cases?

Reply:
The good thing is, changing documentation wouldn't change the current behavior. Gap detection is used in production systems, so people have already identified the best sleep time settings for their projections.

Reply:
I'm with @codeliner here - afaik this is a BC (as unfortunate it may be).